### PR TITLE
Add summaries for some ASP.NET Core namespaces

### PR DESCRIPTION
**Pull request by halter73**
Part of dotnet/aspnetcore#27286.

### CODE REVIEW

**Review by BrennanConroy — APPROVED** [review]

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Http.Features.Authentication.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Http.Features.Authentication">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types defining and implementing authentication <see cref="P:Microsoft.AspNetCore.Http.HttpContext.Features" />.</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/en-us/aspnet/core/fundamentals/request-features">Request Features in ASP.NET Core</see>.</remarks>
```
Comment (Rick-Anderson) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Http.Features.Authentication.xml:4]:
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/fundamentals/request-features">Request Features in ASP.NET Core</see>.</remarks>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Http.Features.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Http.Features">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types related to <see cref="P:Microsoft.AspNetCore.Http.HttpContext.Features" />.</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/en-us/aspnet/core/fundamentals/request-features">Request Features in ASP.NET Core</see>.</remarks>
```
Comment (Rick-Anderson) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Http.Features.xml:4]:
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/fundamentals/request-features">Request Features in ASP.NET Core</see>.</remarks>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Http.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Http">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types for handling HTTP requests and responses.</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-context">Access HttpContext in ASP.NET Core</see>.</remarks>
```
Comment (Rick-Anderson) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Http.xml:4]:
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/fundamentals/http-context">Access HttpContext in ASP.NET Core</see>.</remarks>
```

**Review by Rick-Anderson — APPROVED** [review]
After removing local

**Pull request merged**